drivers/hmis/app/graphql/mutations/ce_referral_step_submit.rb

```diff
@@ -0,0 +1,27 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class CeReferralStepSubmit < CleanBaseMutation
```

Comment:
As discussed there is a need for a "validate" step that dry-runs the submission and notifies the user of the outcome/side-effects.

Either:
* Separate mutation `CeReferralStepValidate` or,
* Use  `CeReferralStepSubmit` pass flag `confirmed:` to match existing form submission pattern. (If `confirmed:false`, then the mutation returns validation warnings about what will happen (if necessary). If `confirmed:true` then the mutation will perform the operation despite validation warnings).

Probably easier for all to use existing pattern (option 2), if it seems workable here.